README.md

```diff
@@ -61,7 +61,7 @@ The package consists of the following PowerShell modules:
 
 ## Installation
 
-We offer several ways of how you can use our free of charge . 
+We offer several ways of how you can use our free of charge Audit Test Automation Package (AuditTAP). 
```

Comment:
I suggest changing the sentence to:
"We offer several ways to use our Audit Test Automation Package (AuditTAP) tool free of charge."